STYLE_GUIDE.md

```diff
@@ -15,23 +15,23 @@ This style guide is a living document and proposed changes may be adopted after
 
 ## File naming conventions
 
-* Source files should contain at most one `program`, `module`, or `submodule`.
-* The filename should match the program or module name and have the file extension `.f90` or `.F90` if preprocessing is required.
+* Source files should contain at most one `program`, `module`, or `submodule`
+* The filename should match the program or module name and have the file extension `.f90` or `.F90` if preprocessing is required
```

Comment:
The first bullet and last bullet points are more or less the same.

Reply:
Good point. I was moving this from another section and didn't realize it was already covered here. Thanks!